src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ThemesController.cs

```diff
@@ -0,0 +1,41 @@
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using GovUk.Education.ExploreEducationStatistics.Admin.Models;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore.Internal;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api
+{
+    [Route("themes")]
```

Comment:
I think the convention so far is to have the routes with a capital letter at the start of a noun i.e. "localhost/api/themes" would be "localhost/api/Themes" - worth checking over the other API endpoints to see if this is right.

Reply:
I'll have a look

Reply:
It taken directly from the controller in most places - I'll make the change